### PR TITLE
removes project notes button and entry from projectNoteTypes array

### PR DESCRIPTION
**Pull request by tillyw**
## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/8720

## Testing
**URL to test:** 
https://deploy-preview-621--atd-moped-main.netlify.app/moped/projects

**Steps to test:**
go to a project, select comments tab

---
#### Ship list
- [x] Code reviewed 
- [x] Product manager approved
- [x] ~Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#)~ not applicable

Screen Shot 2022-03-30 at 6 01 44 PM

### CODE REVIEW

**Comment by frankhereford** [comment]
Hi @tillyw - I edited your comment with a link to the netlify deployment for this PR. Just wanted to give you a heads up in case you wanted us to be testing in another environment - thanks!

File: moped-editor/src/views/projects/projectView/ProjectComments.js [hunk 1]
```diff
@@ -271,7 +270,6 @@ const ProjectComments = props => {
             <CommentButton noteTypeId={0}>All</CommentButton>
             <CommentButton noteTypeId={1}>Internal Notes</CommentButton>
             <CommentButton noteTypeId={2}>Status Updates</CommentButton>
-            <CommentButton noteTypeId={3}>Timeline Notes</CommentButton>
```
Comment (frankhereford) [inline moped-editor/src/views/projects/projectView/ProjectComments.js:274]: I am wondering if there is code that requires this field to be on the page, or at least one with the `noteTypeId={3}`. I made a quick video to demonstrate it, and you can [find it here](https://austininnovation.slack.com/archives/CNUEPKLB1/p1648755790345859).  Thanks!

**Review by frankhereford — CHANGES_REQUESTED** [review]
This isn't so much a request for changes, but more a request to see if the demonstrated behavior is a regression from this deletion or if it's something we had had prior. Thanks!

**Review by mateoclarke — APPROVED** [review]
🚢 💨 ✨

**Review by amenity — APPROVED** [review]
LGTM! ✨🙏

**Review by frankhereford — APPROVED** [review]

**Pull request merged**